plugins/pluginmanagers/lz-n.nix

```
@@ -0,0 +1,106 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+let
+  lzPluginType =
+    with helpers.nixvimTypes;
+    submodule {
+      options = {
```

```suggestion
    submodule {
      freeformType = attrsOf anything;
      options = {
```

To be truly RFC42, and to allow us to omit options for anything unimportant and/or likely to change upstream, we should make this a _freeform_ submodule.

This allows users to define config even if we haven't declared a corresponding option (so long as their config matches either the freeform type or one of our declared options, it is valid).